pkgs/by-name/ar/ares/package.nix

```
@@ -68,31 +72,48 @@ stdenv.mkDerivation (finalAttrs: {
       udev
     ];
 
-  appendRunpaths = [
-    (lib.makeLibraryPath [
-      librashader
-      vulkan-loader
-    ])
+  patches = [
+    ./patches/darwin-build-fixes.patch
```

```suggestion
    ./darwin-build-fixes.patch
```
we don't need to hide the patches in extra directories